tests/jq.test

```diff
@@ -1568,6 +1568,10 @@ strftime("%A, %B %d, %Y")
 1435677542.822351
 "Tuesday, June 30, 2015"
 
+try ["OK", strftime("%Y-%m-%dT%H:%M:%SZ")] catch ["KO", .]
```

Comment:
I am using `try ["OK",` because the test throws an error in the windows CI, and `--run-tests` does not output exceptions to the log, so this is the only way to check what is going on

Reply:
`["KO","strftime/1: unknown system failure"]`; it windows's lib c (lib posix?) is probably stricter than gnu's so it does not allow `mday=0`; i will make sure to at least set `mday=1` so the test is portable
```
The <time.h> header shall declare the tm structure, which shall include at least the following members:

int    tm_sec   Seconds [0,60]. 
int    tm_min   Minutes [0,59]. 
int    tm_hour  Hour [0,23]. 
int    tm_mday  Day of month [1,31]. 
int    tm_mon   Month of year [0,11]. 
int    tm_year  Years since 1900. 
int    tm_wday  Day of week [0,6] (Sunday =0). 
int    tm_yday  Day of year [0,365]. 
int    tm_isdst Daylight Savings flag. 
```